### PR TITLE
Start release 1.12

### PR DESCRIPTION
**Pull request by alvinlin123**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 * [ENHANCEMENT] Blocks storage: Add `-blocks-storage.azure.http.*`, `-alertmanager-storage.azure.http.*`, and `-ruler-storage.azure.http.*` to configure the Azure storage client. #4581
 * [ENHANCEMENT] Optimise memberlist receive path when used as a backing store for rings with a large number of members. #4601
 * [ENHANCEMENT] Add length and limit to labelNameTooLongError and labelValueTooLongError #4595
+* [ENHANCEMENT] Add validation in chunks Redis client's MSet method to avoid panic. #4522
```
Comment (bboreham) [inline CHANGELOG.md:25]: How would an ordinary user know if they are affected by this?
I think it's not supposed to be user-impacting; if you were going to get a panic then something was badly wrong inside the Cortex code.

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,6 +31,7 @@
 * [BUGFIX] Memberlist: fixed corrupted packets when sending compound messages with more than 255 messages or messages bigger than 64KB. #4601
 * [BUGFIX] Query Frontend: If 'LogQueriesLongerThan' is set to < 0, log all queries as described in the docs. #4633
 * [BUGFIX] Distributor: update defaultReplicationStrategy to not fail with extend-write when a single instance is unhealthy. #4636
+* [BUGFIX] Fix data race when stopping chunks memcached client. #4511
```
Comment (bboreham) [inline CHANGELOG.md:34]: This is not expected to be user-impacting either; it was detected in a test.

**Review by bboreham — APPROVED** [review]
OK to go ahead but I don't think it's worth including those extra changes in the changelog.

File: CHANGELOG.md (hunk 1 above)
Comment (alvinlin123) [reply CHANGELOG.md:25]: make sense I'll remove it.

File: CHANGELOG.md (hunk 2 above)
Comment (alvinlin123) [reply CHANGELOG.md:34]: I see, I will remove.

**Pull request merged**